docs/source/starter/introduction_guide.rst

```diff
@@ -99,7 +99,7 @@ Let's first start with the model. In this case, we'll design a 3-layer neural ne
             return x
 
 Notice this is a :doc:`lightning module <../common/lightning_module>` instead of a ``torch.nn.Module``. A LightningModule is
-equivalent to a pure PyTorch Module except it has added functionality. However, you can use it **EXACTLY** the same as you would a PyTorch Module.
+equivalent to a pure PyTorch module except it has added functionality. However, you can use it **exactly** the same as you would a PyTorch module.
```

Comment:
Would maybe keep `Module` as it is a class. Perhaps it should be:
```suggestion
equivalent to a pure PyTorch ``nn.Module`` except it has added functionality. However, you can use it **exactly** the same as you would a PyTorch ``nn.Module``.
```
Otherwise could just have:
```suggestion
equivalent to a pure PyTorch ``Module`` except it has added functionality. However, you can use it **exactly** the same as you would a PyTorch ``Module``.
```